### PR TITLE
Update pull request template to remove fixes keyword, and add a nudge to assign the Fix Inbound label

### PR DESCRIPTION
**Pull request by andrewserong**
This is proposing a small update to the pull request template to remove the "fixes" keyword, based on the suggestion in pciE2j-bQ-p2.

#### Changes proposed in this Pull Request

* Remove the `Fixes` line from the pull request template
* Add a comment to suggest adding the `[Status] Fix Inbound` label to the linked issue

#### Testing instructions

* Check for typos, does the wording seem acceptable?

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -11,4 +11,11 @@ Add as many details as possible to help others reproduce the issue and test the
 
 *
 
-Fixes #
+<!--
+Link a related issue to this PR. If the PR does not immediately resolve the issue,
+for example requires a separate deployment to be rolled out to production, avoid
```
Comment (ramonjd) [inline .github/PULL_REQUEST_TEMPLATE.md]: This is a nice nudge I think.

I've nevertheless just been possessed by a pedant demon, who has coerced me into raising the suggestion that we use a linking verb in this sentence and remove the tautology:

```suggestion
for example, it requires a separate deployment to production, avoid
```

You may ignore this demon.

Another idea for later might be to add a separate PR template, e.g., one for PRs that immediately "fix an issue" and therefore use the "Fixes" keyword. 

https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser

We could then customize the content and labels accordingly if required.

🤷

Comment (andrewserong) [reply .github/PULL_REQUEST_TEMPLATE.md]: Nice, I like it!

**Comment by andrewserong** [comment]
@Automattic/team-calypso this is a tiny change to the PR template, based on suggestions in pciE2j-bQ-p2 — just wanted to get a second opinion on whether this is okay to merge in 🙂

**Review by tyxla — APPROVED** [review]
I this that's an improvement. Personally, I find myself more often deleting the "Fixes #" instead of using it. 

LGTM 👍 🚢

**Pull request merged**